Makefile

```diff
@@ -33,6 +33,13 @@ os_migrate-os_migrate-latest.tar.gz:
 
 # TESTS
 
+test-lint:
```

Comment:
We should add it to test-fast target like: `test-fast: test-lint test-sanity test-unit`. The idea of it was "all checks which don't need OpenStack running".

Reply:
yeahp, done.